From 25518ad91b360915d2ec51dbd2adf228befa825a Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Thu, 23 Jul 2020 17:57:48 +0700
Subject: [PATCH 210/433] media: soc_camera: rcar_vin: Add compatible for
 r8a77470

RZ/G1C has some differences in vin module and need special handling
comparing to other RZ/G1 SoCs. Hence, it should go with different
compatible macro other than RCAR_GEN2.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/media/platform/soc_camera/rcar_vin.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/soc_camera/rcar_vin.c b/drivers/media/platform/soc_camera/rcar_vin.c
index d1aca09..b568e90 100644
--- a/drivers/media/platform/soc_camera/rcar_vin.c
+++ b/drivers/media/platform/soc_camera/rcar_vin.c
@@ -2236,6 +2236,7 @@ static const struct of_device_id rcar_vin_of_table[] = {
 	{ .compatible = "renesas,vin-r8a7790", .data = (void *)RCAR_GEN2 },
 	{ .compatible = "renesas,vin-r8a7779", .data = (void *)RCAR_H1 },
 	{ .compatible = "renesas,vin-r8a7778", .data = (void *)RCAR_M1 },
+	{ .compatible = "renesas,vin-r8a77470", .data = (void *)RZ_G1C },
 	{ .compatible = "renesas,rcar-gen2-vin", .data = (void *)RCAR_GEN2 },
 	{ },
 };
-- 
2.7.4

